src/measure_theory/probability_mass_function.lean

```diff
@@ -402,4 +403,99 @@ rfl
 
 end bernoulli
 
+section outer_measure
```

Comment:
Could you `open measure_theory measure_theory.outer_measure` here, and remove the `measure_theory` and `measure_theory.outer_measure` prefixes in the declarations below (for all cases where the shorter name is not ambiguous)?